test/e2e/systeminternaltls/system_internal_tls_test.go

```diff
@@ -225,12 +219,6 @@ func TestTLSCertificateRotation(t *testing.T) {
 		t.Fatalf("Failed to delete secret %s in system namespace: %v", config.ServingRoutingCertName, err)
 	}
 	checkEndpointState(t, clients, url)
-
-	t.Log("Deleting secret in ingress namespace")
-	if err := clients.KubeClient.CoreV1().Secrets(ingressNS).Delete(context.Background(), config.ServingRoutingCertName, v1.DeleteOptions{}); err != nil {
```

Comment:
It seems the error is:  `system_internal_tls_test.go:231: Failed to delete secret routing-serving-certs in ingress namespace: secrets "routing-serving-certs" not found`. We could skip that error with `if apierrs.IsNotFound(err) ` but I guess we want to keep upstream clean from the downstream assumptions.

Reply:
Right. I wanted to keep upstream clean. Thanks